test/e2e/cypress/e2e/clusters_overview.cy.js

```diff
@@ -122,31 +122,56 @@ context('Clusters Overview', () => {
           .should('have.class', 'fill-red-500');
       });
     });
+  });
 
-    describe('Clusters Tagging', () => {
-      before(() => {
-        cy.removeTagsFromView();
-      });
-      const clustersByMatchingPattern = (pattern) => (clusterName) =>
-        clusterName.includes(pattern);
-      const taggingRules = [
-        ['hana_cluster_1', 'env1'],
-        ['hana_cluster_2', 'env2'],
-        ['hana_cluster_3', 'env3'],
-      ];
-
-      taggingRules.forEach(([pattern, tag]) => {
-        describe(`Add tag '${tag}' to all clusters with '${pattern}' in the cluster name`, () => {
-          availableClusters
-            .map(({ name }) => name)
-            .filter(clustersByMatchingPattern(pattern))
-            .forEach((clusterName) => {
-              it(`should tag cluster '${clusterName}'`, () => {
-                cy.addTagByColumnValue(clusterName, tag);
-              });
+  describe('Clusters Tagging', () => {
+    before(() => {
+      cy.removeTagsFromView();
+    });
+    const clustersByMatchingPattern = (pattern) => (clusterName) =>
+      clusterName.includes(pattern);
+    const taggingRules = [
+      ['hana_cluster_1', 'env1'],
+      ['hana_cluster_2', 'env2'],
+      ['hana_cluster_3', 'env3'],
+    ];
+
+    taggingRules.forEach(([pattern, tag]) => {
+      describe(`Add tag '${tag}' to all clusters with '${pattern}' in the cluster name`, () => {
+        availableClusters
+          .map(({ name }) => name)
+          .filter(clustersByMatchingPattern(pattern))
+          .forEach((clusterName) => {
+            it(`should tag cluster '${clusterName}'`, () => {
+              cy.addTagByColumnValue(clusterName, tag);
             });
-        });
+          });
       });
     });
   });
+
+  describe('Deregistration', () => {
+    const hana_cluster_1 = {
+      name: 'hana_cluster_1',
+      hosts: [
+        {
+          id: '13e8c25c-3180-5a9a-95c8-51ec38e50cfc',
+          name: 'vmhdbdev01',
```

Comment:
I kept them there for reference, so that when reading the test you at least know which host is being deregistered, what do you think? maybe better a comment or nothing at all?